lib/game.rb

```diff
@@ -0,0 +1,75 @@
+require 'pry'
+
+class Game
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],
+    [3,4,5],
+    [6,7,8],
+    [0,3,6],
+    [1,4,7],
+    [2,5,8],
+    [0,4,8],
+    [6,4,2]
+  ]
+
+  def initialize(*args)
+    @player_1 = args[0] || Player::Human.new("X")
+    @player_2 = args[1] || Player::Human.new("O")
+    @board = args[2] || Board.new
+  end
```

Comment:
@aviflombaum Looking for clarification here.  What's the use case for splat and why wouldn't it be as good as explicit arguments?

I included it in this project for two reasons: 
1. I just read about it in POODR and wanted to try it out on my own code because it's a good way to learn things.
2. It extracts the default value assignment outside of the method call, which I thought looked ugly.